inference-engine/src/mkldnn_plugin/mkldnn_exec_network.cpp

```diff
@@ -100,6 +101,7 @@ MKLDNNExecNetwork::MKLDNNExecNetwork(const InferenceEngine::ICNNNetwork &network
         }
     }
 
+    OV_ITT_TASK_NEXT(taskChain, "UnrollPasses");
```

Comment:
As I see UnrollPasses tag cover some scale_shift&prelu passes as well. It's not obvious behavior.

Reply:
The "createConstInputs" counter was added after the "UnrollPasses".